src/LOR/test_models.py

```diff
@@ -0,0 +1,118 @@
+        self.assertEqual(field_label, 'position')
+
+    def test_position_max_len(self):
+        lor = LOR.objects.get(id=1)
+        max_len = lor._meta.get_field('position').max_length
+        self.assertEqual(max_len, 100)
+
+    def test_due_date(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('due_date').verbose_name
+        self.assertEqual(field_label, 'due date')
+
+    def test_writer_email(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('writer_email').verbose_name
+        self.assertEqual(field_label, 'writer email')
+
+    def test_company_name(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('company_name').verbose_name
+        self.assertEqual(field_label, 'company name')
+
+    def test_company_name_max_len(self):
+        lor = LOR.objects.get(id=1)
+        max_len = lor._meta.get_field('company_name').max_length
+        self.assertEqual(max_len, 100)
+
+    def test_company_website(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('company_website').verbose_name
+        self.assertEqual(field_label, 'company website')
+
+    def test_company_email(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('company_email').verbose_name
+        self.assertEqual(field_label, 'company email')
+
+    def test_company_recipients(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('company_recipients').verbose_name
+        self.assertEqual(field_label, 'company recipients')
+
+    def test_company_recipients_max_len(self):
+        lor = LOR.objects.get(id=1)
+        max_len = lor._meta.get_field('company_recipients').max_length
+        self.assertEqual(max_len, 200)
+
+    def test_status(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('status').verbose_name
+        self.assertEqual(field_label, 'status')
+
+    def test_status_max_len(self):
+        lor = LOR.objects.get(id=1)
+        max_len = lor._meta.get_field('status').max_length
+        self.assertEqual(max_len, 20)
+
+    def test_cv(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('cv').verbose_name
+        self.assertEqual(field_label, 'cv')
+
+    def test_resume(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('resume').verbose_name
+        self.assertEqual(field_label, 'resume')
+
+    def test_transcript(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('transcript').verbose_name
+        self.assertEqual(field_label, 'transcript')
+
+    def test_additional_info(self):
+        lor = LOR.objects.get(id=1)
+        field_label = lor._meta.get_field('additional_info').verbose_name
+        self.assertEqual(field_label, 'additional info')
```

Comment:
all of these tests are great and make sure that the correct information is stored and retrieved from the db. In the future if the layout of the LOR is broken down, these tests are important for normal operation when you would want to find a specific letter request and you would have to query the database.